src/std/sugar-test.ss

```diff
@@ -58,4 +59,16 @@
     (check ((is 'a test: eq?) 'a)       => #t)
     (check ((is 2.0) 2.0)               => #t)
     (check ((is "a") "a")               => #t))
+
+   (test-case "test assert! failure message"
+    (def e 'needle)
+    (def l ['stack 'of 'hay])
+    (check-exception (assert! (member e l))
+                     (lambda (e)
+                       (pregexp-match
+                        (string-append
+                         "Assertion failed \"src/std/sugar-test.ss\"@66.31: \\(member e l\\)\n"
```

Comment:
this is a little brittle, can we make it fuzzier? Otherwise we'll have to change the location every tim we edit the file...

Reply:
a regexp should do

Reply:
I do want to test the location... but I suppose one way to make it fuzzier is to allow the line number to be anything, while strictly checking the column number. That way adding more tests in the middle won't affect it.

Reply:
yeah thats about right